eng/pipelines/common/templates/steps/ci-project-build-step.yml

```diff
@@ -34,6 +34,30 @@ parameters:
       - all
 
 steps:
+- task: UseDotNet@2
+  inputs:
+    packageType: sdk
+    version: '8.x'
```

Comment:
Did you mean to use 9.x here?

Reply:
Nope! This is to make sure that the 8.x sdk is installed in the same temp directory on the build agent as the 9.x sdk. This may go away if I can figure out how to make dotnet look in multiple places for sdk versions. It's documented that it should work, but doesn't seem to in this instance. I let it go for the moment while I address some APIs that were marked as obsolete in 9, but I'm getting back to it after that.